extensions/big-data-cluster/src/wizards/create-cluster/createClusterWizard.ts

```diff
@@ -32,10 +34,16 @@ export class CreateClusterWizard extends WizardBase<CreateClusterModel, CreateCl
 		this.setPages([targetClusterTypePage, settingsPage, clusterProfilePage, selectTargetClusterPage, summaryPage]);
 
 		this.wizardObject.generateScriptButton.label = localize('bdc-create.generateScriptsButtonText', 'Generate Scripts');
-		this.wizardObject.generateScriptButton.hidden = true;
+		this.wizardObject.generateScriptButton.hidden = false;
```

Comment:
@alanrenmsft  has this behavior changed on his own branch. Enabling by default for testing purposes. Once Alan merges his change, this button will only be enabled on the summary page.